sdk/src/main/scala/com/horizen/account/certificatesubmitter/AccountCertificateSubmitter.scala

```diff
@@ -46,6 +47,13 @@ class AccountCertificateSubmitter[T <: CertificateData](settings: SidechainSetti
   type MP = AccountMemoryPool
   type PM = AccountBlock
 
+  //override def getUtxoMerkleTreeRoot(referencedEpoch: Int, state: AccountState): Optional[Array[Byte]] = Optional.empty()
+
+  /*
+  override def getWithdrawalRequests(state: AccountState, referencedEpochNumber: Int): Seq[BackwardTransfer] =
+    state.getWithdrawalRequests(referencedEpochNumber).map(request => new BackwardTransfer(request.proposition.bytes, request.valueInZennies))
+  */
+
```

Comment:
changes in this file can be reverted back, seems to be a drawback of the merge

Reply:
Ok, done